shared/util/container.tsx

```diff
@@ -162,6 +164,20 @@ export const useEvent = <Arr extends any[], R>(fn: Fn<Arr, R>): Fn<Arr, R> => {
   )
 }
 
+export const dummyListenerApi = {
```

Comment:
this just lets us use the existing engine helpers w/o much change but we could make this optional and fix it in engine. if we need to use this a lot thats likely better but this is fine for now